engine/src/main/java/org/operaton/bpm/engine/impl/jobexecutor/RuntimeContainerJobExecutor.java

```diff
@@ -63,6 +64,15 @@ public void executeJobs(List<String> jobIds, ProcessEngineImpl processEngine) {
       logRejectedExecution(processEngine, jobIds.size());
       rejectedJobsHandler.jobsRejected(jobIds, processEngine, this);
     }
+
+    if (executorService instanceof JmxManagedThreadPool) {
```

Comment:
Same here with the newer instanceof style, sorry for not catching it in the first review